README.md

```diff
@@ -130,6 +135,9 @@ You can activate the autocomplete following this additional steps:
 - Make sure the script template `DangerFileScript` is active and above the default `Kotlin Script`
 - Apply changes
 - If opening the `Dangerfile.df.kts` the autocomplete and syntax highlighting doesn't work, try to reboot your IDE or open the Dangerfile from your IDE as a single file.
+- If it still doesn't work, go to `Help -> Edit Custom VM Options...` and add `-Dkotlin.script.classpath=<path>/danger-kotlin.jar` (replace the `<path>` with the absolute path to the JAR). Then restart the IDE.
```

Comment:
I actually ran this tool without installing. I got the JAR from the Docker image using
```
docker run --rm ghcr.io/danger/danger-kotlin:1.3.3 gzip --stdout /usr/local/lib/danger/danger-kotlin.jar | gunzip > danger-kotlin.jar
```